### PR TITLE
[CHANGELOG] 4.0.0-alpha.19

### PR DESCRIPTION
**Pull request by DanailH**
As far as I saw there were no breaking changes for this release.

### CODE REVIEW

**Comment by DanailH** [comment]
Once this is merged I'll proceed with the release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,51 @@
...
+- 🎁 Add getRowId prop (#972) @dtassone
+- 🚀 Add streaming delete row api (#980) @dtassone
+- 💅 Fix autoHeight (#940) @oliviertassinari
+- 🙌 Enable the data grid to work under strict mode (#933) @dtassone
+- ⚡️ Add component slots for toolbar and preference panel (#971) @DanailH
+- 🐛 Polish existing features, fix 9 issues.
+
+### @material-ui/x-grid@v4.0.0-alpha.19 / @material-ui/data-grid@v4.0.0-alpha.19
+
+#### Changes
```
Comment (oliviertassinari) [inline CHANGELOG.md:21]: We could remove this subheader. It's really useful to separate breaking changes to the other ones.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,51 @@
...
+- [DataGrid] Add streaming delete row api (#980) @dtassone
+- [DataGrid] Fix autoHeight (#940) @oliviertassinari
+- [DataGrid] Fix column reorder instability (#950) @dtassone
+- [DataGrid] Fix footer visual regression (#932) @dtassone
+- [DataGrid] Fix strict mode issue with apiRef (#933) @dtassone
+- [DataGrid] Fix the accessibility of the column menu (#900) @zj9495
+- [DataGrid] Fix timing guarentee (#981) @oliviertassinari
+- [DataGrid] Fix unstable footer height (#937) @oliviertassinari
+- [DataGrid] Fix usage of the prop-types API (#955) @oliviertassinari
+- [DataGrid] Fix duplicate aria-label (#953) @oliviertassinari@dtassone
```
Comment (oliviertassinari) [inline CHANGELOG.md:34]:
```suggestion
- [DataGrid] Fix duplicate aria-label (#953) @oliviertassinari
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,51 @@
...
+#### Changes
+
+- [DataGrid] Add component slots for toolbar and preference panel (#971) @DanailH
+- [DataGrid] Add getRowId prop (#972) @dtassone
+- [DataGrid] Add streaming delete row api (#980) @dtassone
+- [DataGrid] Fix autoHeight (#940) @oliviertassinari
+- [DataGrid] Fix column reorder instability (#950) @dtassone
+- [DataGrid] Fix footer visual regression (#932) @dtassone
+- [DataGrid] Fix strict mode issue with apiRef (#933) @dtassone
+- [DataGrid] Fix the accessibility of the column menu (#900) @zj9495
```
Comment (oliviertassinari) [inline CHANGELOG.md:30]: We didn't do enough to fix it. 

```suggestion
- [DataGrid] Work on the accessibility of the column menu (#900) @zj9495
```

File: CHANGELOG.md (hunk 1 above)
Comment (DanailH) [reply CHANGELOG.md]: I missed it, there were no breaking changes.

**Review by oliviertassinari — APPROVED** [review]

**Pull request merged**